app/controllers/offline_reservations_controller.rb

```diff
@@ -52,6 +53,12 @@ def update
 
   private
 
+  def flag_ongoing_reservations_as_problem
+    OrderDetail
+      .where(id: @instrument.reservations.ongoing.pluck(:order_detail_id))
+      .update_all(problem: true)
```

Comment:
The `problem` field is a field that gets updated as part of a `before_save` callback. We should not be setting it directly. Looking at that code, it looks like we need to mark the order as complete. Since it'll be missing the `actual_end_at`, it should get itself into problem status. I think this is what we do when we auto_expire reservations.